app/assets/javascripts/discourse/app/components/topic-title.js

```diff
@@ -9,7 +9,7 @@ export function addTopicTitleDecorator(decorator) {
 }
 
 export function resetTopicTitleDecorators() {
-  topicTitleDecorators = [];
+  topicTitleDecorators.length = 0;
```

Comment:
Is there somewhere I can read more about this? I'm not quite sure why this is correct as compared to just reassigning the variable.

Reply:
Ahh the original might still be referenced somewhere preventing it from being GC-ed?

Reply:
Yup, here's a comment by @martin-brennan that explains it:

https://github.com/discourse/discourse/blob/2d58996a3bc1e283303305eb2d439c93f9cdbcb3/app/assets/javascripts/discourse/app/components/composer-editor.js#L63-L69

And worse than it not being GC-ed, all places that imported a variable like that were stuck using the first version of it - that was the source of some test leaks I believe